### PR TITLE
Unify cipher and plain

### LINKED ISSUE #16001: Unify encrypt test cases.
In encrypt test, we have two config.
`cipher` and `plain` 
They are almost same but for `queryWithCipherColumn: false/true`
So the cases should be same for each config.
I want to add test cases if any config is missing cases.

### PR DESCRIPTION
**Pull request by tuichenchuxin**
Fix #16001 
- Unify cipher and plain test
- Add test case

### CODE REVIEW

**Review by strongduanmu — CHANGES_REQUESTED** [review]
@tuichenchuxin Can you take a look at ci exception?

**Comment by tuichenchuxin** [comment]
> @tuichenchuxin Can you take a look at ci exception?

Thank you. Done.

File: shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/encrypt/case/query-with-plain/dml/select/select_subquery.xml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+<!--        <input sql="SELECT u.amount, u.password, o.certificate_number FROM (SELECT certificate_number FROM t_account_bak) o, t_account u WHERE o.certificate_number=u.certificate_number AND u.password=?" parameters="1" />-->
+<!--        <output sql="SELECT u.cipher_amount AS amount, u.cipher_password AS password, o.plain_certificate_number AS certificate_number FROM (SELECT cipher_certificate_number, assisted_query_certificate_number, plain_certificate_number FROM t_account_bak) o, t_account u WHERE o.assisted_query_certificate_number=u.assisted_query_certificate_number AND u.assisted_query_password=?" parameters="assisted_query_1" />-->
+<!--    </rewrite-assertion>-->
+    
+    <rewrite-assertion id="select_not_nested_subquery_in_table_segment_alias_for_plain" db-types="MySQL">
+        <input sql="SELECT o.certificate_number FROM (SELECT a.certificate_number FROM t_account a) o" />
+        <output sql="SELECT o.cipher_certificate_number AS certificate_number FROM (SELECT a.cipher_certificate_number, a.assisted_query_certificate_number FROM t_account a) o" />
+    </rewrite-assertion>
+    
+<!--    TODO FIX ME!-->
```
Comment (strongduanmu) [inline shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/encrypt/case/query-with-plain/dml/select/select_subquery.xml:43]: @tuichenchuxin Why comment these rewrite test case?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/encrypt/case/query-with-plain/dml/select/select_subquery.xml (hunk 1 above)
Comment (tuichenchuxin) [reply shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/encrypt/case/query-with-plain/dml/select/select_subquery.xml:43]: These test case is the new case, and something is wrong with encrypt, this needs to be fixed.

Comment (strongduanmu) [reply shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/encrypt/case/query-with-plain/dml/select/select_subquery.xml:43]: > encrypt

Got it.

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**